runtime/vm/stringhelpers.cpp

```diff
@@ -450,11 +451,14 @@ verifyQualifiedName(J9VMThread *vmThread, j9object_t string)
 	 */
 	if (arity < 0) {
 		return CLASSNAME_INVALID;
-	} else if (0 == arity) {
-		return CLASSNAME_VALID_NON_ARRARY;
-	} else {
-		return CLASSNAME_VALID_ARRARY;
 	}
+
+	result = (0 == arity) ? CLASSNAME_VALID_NON_ARRARY : CLASSNAME_VALID_ARRARY;
+	if (J9_ARE_ANY_BITS_SET(expectedReturnCode, result)) {
```

Comment:
Though it doesn't matter in practice, the parameters in the macro should really be reversed.

Reply:
Agreed and fixed as suggested above.